pages/assess/dashboards_and_queries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Queries and Dashboards
+layout: pages
+parent: Assess
+keywords: dashboard, query, collection, organize, refine
+last_updated: September 2025
+summary: "Create complex queries, build dashboards of queries and share them with your team"
+permalink: dashboards-and-queries.html
+---
+
+In Tidal Accelerator, you can save complex queries built with our advanced filtering system to your inventory dashboard. Queries can include filters, as well as search values and tags. 
+
+Say for example you'd like to keep track of all applications which are in the 'production' environment, their transition plan is complete but they have not yet been migrated. You can save this as a query, so that you can apply this query with a single click whenever you'd like, rather than needing to manually filter for these values each time you want to work with this subset of applications.
```

Comment:
```suggestion
For example, you may want to track all applications in the 'production' environment that have a complete transition plan but have not yet been migrated. You can save this as a query, so that you can apply this query with a single click whenever you'd like, rather than needing to manually filter for these values each time you want to work with this subset of applications.
```